src/ucs/time/timerq.c

```diff
@@ -32,6 +32,7 @@ void ucs_timerq_cleanup(ucs_timer_queue_t *timerq)
         ucs_warn("timer queue with %d timers being destroyed", timerq->num_timers);
     }
     ucs_free(timerq->timers);
+    ucs_spinlock_destroy(&timerq->lock);
```

Comment:
Do we need `ucs_spinlock_destroy` in this part?
`pthread_spin_destroy` doesn't exist in the original code.

Reply:
yes, we need it